cluster/gce/util.sh

```diff
@@ -475,32 +475,48 @@ function create-node-template() {
     fi
   fi
 
-  local attempt=1
+  local gcloud="gcloud"
+
   local preemptible_minions=""
   if [[ "${PREEMPTIBLE_NODE}" == "true" ]]; then
     preemptible_minions="--preemptible --maintenance-policy TERMINATE"
   fi
+
   local local_ssds=""
   if [ ! -z ${NODE_LOCAL_SSDS+x} ]; then
       for i in $(seq ${NODE_LOCAL_SSDS}); do
           local_ssds="$local_ssds--local-ssd=interface=SCSI "
       done
   fi
+
+  local network="--network ${NETWORK} --can-ip-forward"
+  if [[ ${ENABLE_IP_ALIASES} = "true" ]]; then
```

Comment:
I'm seeing this same block over and over in your PR. What if we make a function in util.sh that generates the network flag?